packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -671,6 +664,8 @@ void main() {
             selectionColor: selectionColor,
             cursorColor: cursorColor,
             child: Column(
+              // TODO(bleroux): investigate why the test fail without setting the mainAxisAlignment.
```

Comment:
That is interesting, was this test working with no changes before this PR? I don't see why this PR would cause that to change.

Reply:
Yes, I can confirm that this test is working on master without this change.
With this PR, I just found that another solution to make this test green is to add a call to `pumpAndSettle()` on line 692, so there is probably an animation going on.
Do you think it is better to keep this TODO or to replace it with the call to pumpAndSettle? There are dozens of pumpAndSettle in this test file, so I think it makes sense to use one here, the remaining question is why was it not necesseray before? (it might be related to the toolbar content being longer than before).

Reply:
Adding a `pumpAndSettle` sounds reasonable. I tried the pr out locally and found that if I change the logic in `bool get shareEnabled` to the code below the test passes. I'm not at all sure why this works. I think we should we keep the switch statement and add a `pumpAndSettle` since we might support other platforms.

```dart
    if (defaultTargetPlatform != TargetPlatform.iOS || defaultTargetPlatform != TargetPlatform.android) {
      return false;
    }

    return !widget.obscureText
        && !textEditingValue.selection.isCollapsed
        && textEditingValue.selection.textInside(textEditingValue.text).trim() != '';
```